src/Microsoft.Graph/Generated/ediscovery/model/AddToReviewSetOperation.cs

```diff
@@ -20,14 +20,14 @@ namespace Microsoft.Graph.Ediscovery
     public partial class AddToReviewSetOperation : CaseOperation
     {
     
-		///<summary>
-		/// The AddToReviewSetOperation constructor
-		///</summary>
+        ///<summary>
+        /// The AddToReviewSetOperation constructor
```

Comment:
Why does this one still have its constructor?

Reply:
Hey @baywet, There is some explanation in the description of generation PR [here](https://github.com/microsoftgraph/MSGraph-SDK-Code-Generator/pull/534).  

Essentially, because of scenarios encountered in past issues (e.g. https://github.com/microsoftgraph/msgraph-sdk-dotnet-core/issues/36#issuecomment-529597352 and https://github.com/microsoftgraph/msgraph-sdk-dotnet/issues/586) removing the constructor in all scenarios will result in API errors when the model in question needs type disambiguation if the type is be referenced as its base type (as a property in another type or an action) or the base is abstract.

In this case, since we can post to the collection through the base type [here](https://github.com/microsoftgraph/msgraph-beta-sdk-dotnet/blob/13cea3aaa6e7dd04098885b6d6883d445b81692e/src/Microsoft.Graph/Generated/ediscovery/requests/CaseOperationsCollectionRequest.cs#L42), if the user creates/posts an instance of the derived type i.e. AddToReviewSetOperation, we will need to disambiguate the type by setting the odata.type in this scenario.

Reply:
Thanks for the context. I'm not finished going through the generator PR just yet. The only thing I'm worried about is the fact we specify the odata type for some classes and not other might have an unreliable result:

It'll work for collections of mixed types (i.e. collection maps to the parent abstract and can have any of the child types). But it might fail if the same child type is used somewhere else and we force the type because of the CTOR.

The answer to the question "when should a client application set the odata type?" is not clearly defined (at least for me) and it'd be nice to get a definitive answer from OData owners before moving to implementation.

Reply:
@baywet 
I had discussions with @darrelmiller about this. 
There should be nothing wrong with setting the odata.type value and ideally the various apis should have no problem with the value being set at any time. However, some workloads are do throw errors when it is set, and it should ideally be fixed by the various workloads. This is going to be followed up [here](https://github.com/microsoftgraph/sample-graph-api/issues/49).

However, when the original requests were raised for us to set them in the first place (e.g [here](https://github.com/microsoftgraph/msgraph-sdk-dotnet/issues/586) and [here](https://github.com/microsoftgraph/msgraph-sdk-dotnet-core/issues/36#issuecomment-529597352)), we realized unsetting all the values will break the experience of existing users in those scenarios as well. 
With this work, we also realized that the API's that do need type disambiguation (e.g. when posting to collections mixed types or when a property's base is referenced) do not have issues with this as seen from the list of issues raised. Hence, we continue to support them in these scenarios.

Reply:
thanks for the detailed explanation! I also reached out to him and the conversation resulted in this: to fully fix this issue end to end it needs to be fixed both on the SDK but also on some workloads. We're doing all we can on the SDK side at this point.